src/OrbitBase/include/OrbitBase/ProcessId.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_LINUX_PROC_H_
+#define ORBIT_BASE_LINUX_PROC_H_
+
+#include <sys/types.h>
+
+#include <vector>
+
+namespace orbit_base {
+
+#if defined(__linux)
+std::vector<pid_t> GetAllPids();
```

Comment:
Add short comment of what these functions do.

I am also thinking if they should be renamed to Find* of Fetch*, probably fine as it is.

Reply:
"Find" is inaccurate, it's not a search: "Fetch", "Retrieve" or "List" are fine by me (but the current "Get" is also fine by me).

Reply:
I'm fine with 'Get'. Added comments.